plugins/window_size/macos/Classes/FLEWindowSizePlugin.m

```diff
@@ -71,6 +76,46 @@ - (NSArray *)platformChannelRepresentationForFrame:(NSRect)frame;
 
 @end
 
+/**
+ * Converts the channel representation for unconstrained minimum size `-1` to Cocoa's specific minimum size of `0`.
+ */
+static double _setWindowMinSizeUnconstrainedCheck(double size){
+    if (size != -1) {
+      return size;
+    }
+    return 0;
+}
+
+/**
+ * Converts the channel representation for unconstrained maximum size `-1` to Cocoa's specific maximum size of `FLT_MAX`.
+ */
+static double _setWindowMaxSizeUnconstrainedCheck(double size) {
```

Comment:
Same comments about naming (and potentially collapsing using `?:`) here as on the Dart side.

Also, style-wise, No `_`, and the first letter needs to be capitalized.